android_app/app/src/main/java/org/miniproject/safesweeper/MainActivity.java

```diff
@@ -53,8 +55,27 @@ protected void onCreate(Bundle savedInstanceState) {
         throttleText = (TextView) findViewById(R.id.throttleText);
         steeringText = (TextView) findViewById(R.id.steeringText);
         connectionTextView = (TextView) findViewById(R.id.connectionTextView);
+        steeringToggle = (ToggleButton) findViewById(R.id.steeringToggle);
 
         new ConnectBT().execute();
+        //toggle switch to go between manual steering and automatic steering, starts with manual
+        steeringToggle.setOnClickListener(new View.OnClickListener() {
```

Comment:
Should this be here or in the onPostExecute of the bluetooth AsyncTask?

Reply:
I really don't know! I can move it if you feel like it doesn't belong here.

Reply:
We can look at this on monday.